client/extensions/woocommerce/woocommerce-services/views/service-settings/shipping-services/group.js

```diff
@@ -108,7 +108,7 @@ const ShippingServiceGroup = props => {
 
 ShippingServiceGroup.propTypes = {
 	title: PropTypes.string.isRequired,
-	deliveryEstimate: PropTypes.string,
+	deliveryEstimate: PropTypes.oneOfType( [ PropTypes.string, PropTypes.bool ] ),
```

Comment:
I completely agree with you, so I found the only component that uses `ShippingServiceGroup` and changed the default `deliveryEstimate` to an empty string instead of `false`.